mud-contracts/world/src/modules/smart-gate/systems/SmartGate.sol

```diff
@@ -153,22 +153,22 @@ contract SmartGate is EveSystem, AccessModified {
   }
 
   /**
-   * @notice Configure Smart Turret
-   * @param smartGateId is smart object id of the Smart Turret
-   * @param systemId is the system id of the Smart Turret logic
+   * @notice Configure Smart Gate
+   * @param smartObjectId is smartObjectId of the Smart Gate
+   * @param systemId is the system id of the Smart Gate logic
    */
   function configureSmartGate(
-    uint256 smartGateId,
+    uint256 smartObjectId,
     ResourceId systemId
-  ) public onlyAdminOrObjectOwner(smartGateId) hookable(smartGateId, _systemId()) {
-    SmartGateConfigTable.setSystemId(_namespace().smartGateConfigTableId(), smartGateId, systemId);
+  ) public onlyAdminOrObjectOwner(smartObjectId) hookable(smartObjectId, _systemId()) {
+    SmartGateConfigTable.setSystemId(_namespace().smartGateConfigTableId(), smartObjectId, systemId);
   }
 
   /**
    * @notice view function for smart gates which is linked
-   * @param characterId is of the id of the character
-   * @param sourceGateId is the id of the source gate
-   * @param destinationGateId is the id of the destination gate
+   * @param characterId is of the smartObjectId of the character
```

Comment:
Should we treat all ids as smart object ids? So a character id will not be simply the character's integer as uint256?

Reply:
I am afraid this might be a huge change for the game play. May be next iteration ?

Reply:
Yeah I think so. I think we should for now just change everything that is being generated as smart object id **and** are primary keys of said objects, to be called `smartObjectId`.

Reply:
We cant do this in some cases eg: 
```
function linkSmartGates(uint256 sourceGateId, uint256 destinationGateId)
```
Thought both are the smartObjectIds we cannot have the same naming. So in some places its okay to have descriptive names I suppose. WDYT ?

Reply:
Yes exactly. In this case these are not primary keys, more like foreign keys.

I guess for "primary keys" we should strive to use `smartObjectId` (usually for primary keys I just use `id` but since we've been using `smartObjectId` thus far I think that's alright).

Reply:
cool, so `characterId` to `smartObjectId`  change in this iteration or next ?

Reply:
I think `characterId` can keep its name and still be just a normal integer (not of the type `smartObjectId`).